src/main/resources/portal.properties.EXAMPLE

```diff
@@ -202,3 +202,13 @@ ucsc.build=hg19
 
 # default view in oncoprint (sample, patient (default))
 oncoprint.defaultview=patient
+
+# labels to be displayed in oncoprint "Mutation color" menu for custom annotation of driver and passenger mutations in the oncoprint.
+# Set any of these properties to enable the respective menu options in oncoprint:
+# oncoprint.custom_driver_annotation.binary.menu_label=Custom driver annotation
+# oncoprint.custom_driver_annotation.tiers.menu_label=Custom driver tiers
+
+# set this to false to disable the automatic selection of the custom driver and passenger mutations annotation in the oncoprint 
+# (instead of OncoKB and Hotspots). By default this is true. Setting this to false will select OncoKB and Hotspots as the default 
+# annotation source instead.
+# oncoprint.custom_driver_annotation.default=false
```

Comment:
`# oncoprint.custom_driver_annotation.default=false`
If this enabled, this option disables OncoKB and Hotspots, and ONLY enables the Putative_Driver (Binary) checkbox. The class checkbox cannot be enabled, because it is not known which class is preferred. The name and documentation should make this clear.